pyproject.toml

```diff
@@ -37,6 +37,7 @@ grpcio = ">=1.56.0"
 plotly = "^5.15.0"
 pyarrow = ">=4.0.0"
 grpcio-status = ">=1.56.0"
+pyfakefs = "^5.2.4"
```

Comment:
@xinrong-meng since it's just for benchmark, we can move the new one to [tool.poetry.dev-dependencies]